schema/schema.hh

```diff
@@ -916,11 +921,11 @@ public:
      * table's id and dropped columns. The dropped columns are present in column definitions and also the `ALTER DROP`
      * statement (and `ALTER ADD` if the column has been re-added) to the description.
      */
-    cql3::description describe(const replica::database& db, cql3::describe_option) const;
+    cql3::description describe(schema_describe_helper& helper, cql3::describe_option) const;
```

Comment:
Isn't it too strong requirement, that the caller should have helper in stable location?
Also, the database_schema_describe_helper contains just one pointer (well, a reference), so if passing the helper by value here, the size of copied around argument will be the same, but accessing it would cost one dereference less

Reply:
You can't pass it by value, since the runtime type is unknown.

Reply:
But, a const wouldn't hurt.

Reply:
> You can't pass it by value, since the runtime type is unknown.

Ah, indeed. I withdraw my suggestion